tests/test_helper.go

```diff
@@ -0,0 +1,337 @@
+package tests
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"strings"
+
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+)
+
+// common constants
+const (
+	E2ETestObjectName  = "e2e-object"
+	MigrationNamespace = "openshift-migration"
```

Comment:
Please use the constant values provided by the API wherever possible https://github.com/konveyor/mig-controller/blob/f97cd1e001b122d6fbc73b95620b6a27f64bacb1/pkg/apis/migration/v1alpha1/resource.go#L18-L19